### ISSUE TITLE
Failed test 71?

### ISSUE THREAD

**Issue by juhop** [VowpalWabbit/vowpal_wabbit#397]
Running 'make test' I get consistently this test 71 failure.  

---

... 
...
RunTests: test 69: stderr OK
RunTests: test 69: predict OK
RunTests: test 70: stderr OK
RunTests: test 70: predict OK
--- diff -u --minimal train-sets/ref/ksvm_train.rbf.stderr stderr.tmp
--- train-sets/ref/ksvm_train.rbf.stderr    2014-09-07 23:11:51.458300953 +0300
+++ stderr.tmp  2014-09-09 17:55:40.404544503 +0300
@@ -18,7 +18,7 @@
 1.003068   0.928672            4         4.0  -1.0000  -0.1453      134
 0.944367   0.885665            8         8.0  -1.0000  -0.2848      145
 0.929273   0.914179           16        16.0   1.0000  -0.4176       23
-0.917699   0.906124           32        32.0  -1.0000  -0.3805       31
+0.917699   0.906125           32        32.0  -1.0000  -0.3805       31
 0.904987   0.892275           64        64.0  -1.0000  -0.5601       60
 0.924562   0.944137          128       128.0   1.0000  -0.1724      105

@@ -26,12 +26,12 @@
 number of examples = 250
 weighted example sum = 250
 weighted label sum = -22
-average loss = 0.888308
+average loss = 0.888046
 best constant = -0.088
 total feature number = 19870
 Num support = 250
-Number of kernel evaluations = 192975 Number of cache queries = 172366
-Total loss = 222.077
+Number of kernel evaluations = 194681 Number of cache queries = 170561
+Total loss = 222.011
 Done freeing model
 Done freeing kernel params
 Done with finish 
RunTests: test 71: FAILED: ref(train-sets/ref/ksvm_train.rbf.stderr) != stderr(stderr.tmp)
    cmd: /usr/bin/timeout 20 ../vowpalwabbit/vw --ksvm --l2 1 --reprocess 5 -b 18 --kernel rbf -p train-sets/ref/ksvm_train.rbf.predict -d train-sets/rcv1_smaller.dat
make: **\* [test] Error 1
